tools/js-sdk-release-tools/src/mlc/clientGenerator/utils/typeSpecUtils.ts

```diff
@@ -14,8 +15,11 @@ export async function generateTypeScriptCodeFromTypeSpec(
     options: ModularClientPackageOptions,
     originalVersion: string | undefined,
     packageDirectory: string
-): Promise<void> {
+): Promise<void> {    
     const tspConfigPath = join(options.typeSpecDirectory, 'tspconfig.yaml');
+    logger.info(`mlc:tspConfigPath: ${tspConfigPath}`);   
+    updateApiVersionInTspConfig(tspConfigPath, options.apiVersion);
```

Comment:
could you move it to https://github.com/Azure/azure-sdk-tools/blob/main/tools/js-sdk-release-tools/src/autoGenerateInPipeline.ts , both rlc and mlc need it